package.json

```diff
@@ -7,13 +7,13 @@
   "license": "MPL-2.0",
   "homepage": "https://github.com/openmrs/openmrs-esm-patient-registration#readme",
   "scripts": {
-    "start": "openmrs debug --importmap @",
```

Comment:
The preference should be `openmrs debug --run-project`. The `webpack-dev-server` may still make sense. I'd leave it in as `start:standalone` or similar (personally, I always use `npx webpack-dev-server` for this).